src/lemlib/chassis/chassis.cpp

```diff
@@ -18,9 +18,9 @@ namespace lemlib {
  *
  * This function is a wrapper for the Odometry::setPose() function
  */
-void Chassis::setPose(float x, float y, float theta, bool radians) {
+void Chassis::setPose(Length x, Length y, Angle theta) {
     Pose pose(x, y, theta);
```

Comment:
```suggestion
    odom->setPose(x, y, theta);
```
Change pose directly